### PR TITLE
Only show updates if manifest version was updated

### LINKED ISSUE #89: Make the updater only show plugin updates if the manifest version is changed
Per [this poll](https://discord.com/channels/1000926524452647132/1000955961487400971/1007061218357432391), the plugin/theme updater should only show updates for a plugin if the manifest version is different than the installed version's.

### PR DESCRIPTION
**Pull request by elementemerald**
Closes #89

### CODE REVIEW

**Comment by asportnoy** [comment]
Still need to test/review this, but **do not merge this yet**. We need to announce to developers and give them sufficient notice. You can still test and review though.

File: src/fake_node_modules/powercord/entities/Updatable.js [hunk 1]
```diff
@@ -74,6 +74,25 @@ class Updatable extends Events {
     return commits;
   }
 
+  async _getUpdateVersion () {
+    const latest = await this._getUpdateCommits();
+    if (latest[0]) {
+      try {
+        const gitManifest = await PowercordNative.exec(`git show ${latest[0].id}:manifest.json`, {
```
Comment (asportnoy) [inline src/fake_node_modules/powercord/entities/Updatable.js:81]: Themes use `powercord_manifest.json` (same JSON key)

**Review by asportnoy — CHANGES_REQUESTED** [review]
Small issue

**Review by asportnoy — CHANGES_REQUESTED** [review]
I think having the same code repeated twice like this isn't ideal. Maybe move it back to updatable and add either use `entity instanceof Plugin` or add a `type` property to each to determine which file to use.

File: src/fake_node_modules/powercord/entities/Updatable.js [hunk 2]
```diff
@@ -11,9 +11,10 @@ const TIMEOUT = 10e3;
  * @property {String} updateIdentifier
  */
 class Updatable extends Events {
-  constructor (basePath, entityID, updateIdentifier) {
+  constructor (basePath, entityType, entityID, updateIdentifier) {
```
Comment (asportnoy) [inline src/fake_node_modules/powercord/entities/Updatable.js:14]: Add this parameter to `src/Powercord/index.js#48`, otherwise LGTM

**Review by asportnoy — CHANGES_REQUESTED** [review]

File: src/Powercord/index.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ let coremods;
  */
 class Powercord extends Updatable {
   constructor () {
-    super(join(__dirname, '..', '..'), '', 'powercord');
+    super(join(__dirname, '..', '..'), '', 'plugin', 'powercord');
```
Comment (asportnoy) [inline src/Powercord/index.js:48]: I think type `plugin` isn't appropriate here, maybe `base`?
```suggestion
    super(join(__dirname, '..', '..'), '', 'base', 'powercord');
```

**Review by asportnoy — APPROVED** [review]

**Comment by asportnoy** [comment]
Will give notice in Discord and merge in a few days

**Review by lexisother — APPROVED** [review]
Yop.

**Pull request merged**